### PR TITLE
Added `shift + space` shortcut

### PR DESCRIPTION
**Pull request by vxsx**
New shortcut toggles structureboard if you hover over a plugin and
then highlights the plugin in the appropriate place.

As part of this commit, `Mousetrap` library is added since in
future we want to add more keyboard shortcuts to the CMS.

Short screencast: http://d.pr/v/1f5DG

### CODE REVIEW

File: cms/static/cms/js/modules/cms.plugins.js [hunk 1]
```diff
@@ -1863,6 +1841,74 @@ Plugin._isContainingMultiplePlugins = function _isContainingMultiplePlugins(node
...
+    // start animation
+    tpl.fadeOut(SUCCESS_TIMEOUT, function () {
+        $(this).remove();
+        el.removeClass('cms-draggable-success');
+    });
+    // make sure structurboard gets updated after success
+    $(Helpers._getWindow()).trigger('resize.sideframe');
+};
+
+Plugin._highlightPluginContent = function _highlightPluginContent(pluginId) {
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: missing jsdocs

**Comment by FinalAngel** [comment]
👍

File: cms/static/cms/js/modules/cms.plugins.js (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/js/modules/cms.plugins.js]: added

**Comment by vxsx** [comment]
screen shot 2016-08-11 at 13 52 35

**Pull request merged**